.ci/test_r_package.sh

```diff
@@ -7,6 +7,10 @@ mkdir -p $R_LIB_PATH
 export R_LIBS=$R_LIB_PATH
 export PATH="$R_LIB_PATH/R/bin:$PATH"
 
+# hack to get around this:
+# https://stat.ethz.ch/pipermail/r-package-devel/2020q3/005930.html
+export _R_CHECK_SYSTEM_CLOCK_=0
```

Comment:
@StrikerRUS I'm proposing adding this hack for now, for the new NOTE about R not knowing what time it is. I think that's better than just increasing the allowed number of NOTEs.

When this issue is resolved, if we still have ALLOWED_CHECK_NOTES +1, then some other note could sneak past us in CI.

Reply:
Agree!